tests/test_core/test_alignment.py

```diff
@@ -1510,6 +1510,15 @@ def test_construction(self):
         seq2 = make_seq("AC", name="seq2")
         coll = SequenceCollection(data=[seq1, seq2])
 
+    def test_write_to_json(self):
+        # test writing to json file
+        aln = self.Class([("a", "AAAA"), ("b", "TTTT"), ("c", "CCCC")])
+        fn = mktemp(suffix=".json")
+        aln.write(fn)
+        result = load_unaligned_seqs(fn)
```

Comment:
Don't use `load_unaligned_seqs` for this. Directly use the `json` module, since you need to explicitly check that you wrote json content. I suggest the following:
```python
got = json.load(fn)
self.assertEqual(got, aln.to_rich_dict()) # that dict method is what's called by `to_json()`
```